### PR TITLE
copy all things from manifest into ENV.json so we can get rid of it

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/runway 

### Risks
 - None

### CODE REVIEW

**Comment by gabetax** [comment]
:+1:

File: plugins/env/lib/samson_env/samson_plugin.rb [hunk 1]
```diff
@@ -38,12 +38,17 @@ def write_dotenv_file(base_file, groups)
 
     # writes a proprietary .json file with a env hash for each deploy group
     def write_env_json_file(env_json, manifest_json, groups)
-      return unless File.exist?(env_json)
-      json = JSON.load(File.read(env_json))
-      File.unlink(env_json)
+      return unless File.exist?(manifest_json)
```
Comment (gabetax) [inline plugins/env/lib/samson_env/samson_plugin.rb]: This method is starting to grow a little long. May be worth refactoring if it gets any larger. Were you planning on using the zendesk_service_manifest gem for this down the road?

Comment (grosser) [inline plugins/env/lib/samson_env/samson_plugin.rb]: not a big fan of this gem ... lots of code for very little functionality + it's private ...

Comment (grosser) [inline plugins/env/lib/samson_env/samson_plugin.rb]: also the env code will get deleted once rosetta+account are deployed with their update, so it should get less complex soon :)

**Comment by grosser** [comment]
confirmed on staging, here we go :)
https://samsontest.zende.sk/projects/rosetta/deploys/2360

**Pull request merged**